tests/EventTest.php

```diff
@@ -274,6 +362,25 @@ public function testGetExtraContext(): void
         $this->assertInstanceOf(Context::class, $event->getExtraContext());
     }
 
+    public function testGetContexts(): void
```

Comment:
I don't think that this test is useful because using typehints we are guaranteed that the returned value is of the expected type